mindjunkies/courses/forms.py

```
@@ -2,47 +2,50 @@
 from django.forms import inlineformset_factory
 from taggit.forms import TagWidget
 
-from .models import Course, CourseInfo, CourseToken, Rating
-
+from .models import Course, CourseInfo, Rating
 
 text_area = "form-input mt-1 block w-full p-2 border border-gray-300 rounded-md focus:ring-indigo-500 focus:border-indigo-500"
```

Ruff (0.8.2)

Line too long (126 > 119)

(E501)